kin-sdk-core/src/androidTest/java/kin/sdk/core/KinAccountTest.java

```diff
@@ -145,6 +145,27 @@ public void sendTransactionSync_wrongPassphrase() {
         }
     }
 
+    @Test
+    public void sendTransactionSync_SecondTimeFails() throws Exception {
+        KinAccount senderAccount = importedAccounts.get(0);
+        Balance senderBalance = senderAccount.getBalanceSync();
+        BigDecimal amountToSend = new BigDecimal(10);
+
+        senderAccount.sendTransactionSync(kinAccount.getPublicAddress(), PASSPHRASE, amountToSend);
+
+        Balance kinAccountBalance = kinAccount.getBalanceSync();
+        assertTrue(kinAccountBalance.value(0).equals("10"));
+
+        Balance afterBalance = senderAccount.getBalanceSync();
+        assertTrue((senderBalance.value().subtract(amountToSend).compareTo(afterBalance.value())) == 0);
+
+        try {
+            senderAccount.sendTransactionSync(kinAccount.getPublicAddress(), "wongPassphrase", new BigDecimal(1));
```

Comment:
use an empty string for a passphrase 
write a second almost identical test sends null the second time

Reply:
added 👍